node/jaeger/src/spans.rs

```diff
@@ -283,6 +291,28 @@ impl Span {
 		}
 	}
 
+	/// Derive a child span with a traceID string tag set to the decimal representation of the CandidateHash.
+	pub fn child_with_trace_id(&self, name: &'static str, trace_id: CandidateHash) -> Self {
+		let mut span = match self {
+			Self::Enabled(inner) => Self::Enabled(inner.child(name)),
+			Self::Disabled => Self::Disabled,
+		};
+		span.add_string_tag("traceID", hash_to_trace_identifier(trace_id.0).to_string());
+		span
+	}
+
+	/// Derive a child span with a start time set to the current system time using mick_jaeger::StartTime::now().
+	/// This is useful for spans that are created after the parent span has started.
+	pub fn child_with_start_time(&self, name: &'static str) -> Self {
```

Comment:
Yes it does, but it is not clear why since it does not invoke the inner method in the mick-jaeger crate. This is not really used anyways so can easily remove this.